modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -9,7 +9,10 @@
 	--n-accordion-item-title-flex-grow: initial;
 	--n-accordion-item-title-space-between: 0px;
 	--n-accordion-item-title-distance-from-content: 0px;
-	--n-accordion-padding: 10px;
+	--n-accordion-padding-block-start: 10px;
```

Comment:
Just for clarity, what is the problem with using the shorthand directly?
I am happy to use the shorthand, because it is quicker.
Not sure what Rami preferes.

Reply:
Padding here is set on all 4 sides by the NA padding control and would become incompatible with the intended LTR / RTL tweak we're introducing if I revert this change.

Reply:
Reverting, Moved this to a separate PR.